packages/@o3r/analytics/plugins/webpack/build-metrics/build-stats.plugin.ts

```diff
@@ -236,18 +235,17 @@ export class BuildStatsPlugin implements WebpackPluginInstance {
     }
   }
 
+  // eslint-disable-next-line unicorn/consistent-function-scoping -- keep for readability
```

Comment:
it's weird to justify disabling this rule with `keep for readability` when the purpose of the rule is to improve readability